src/wp-includes/class-wp-dependencies.php

```diff
@@ -490,4 +490,40 @@ public function set_group( $handle, $recursion, $group ) {
 
 		return true;
 	}
+
+	/**
+	 * Get etag header for cache validation.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string[] $load Array of script or style handles to load.
```

Comment:
```suggestion
	 * @since 6.7.0
	 *
	 * @global string $wp_version The WordPress version string.
	 *
	 * @param string[] $load Array of script or style handles to load.
```

Add missing global docs.